Modules/Sources/Networking/Remote/PaymentRemote.swift

```diff
@@ -28,6 +28,7 @@ public protocol PaymentRemoteProtocol {
 
 /// WPCOM Payment Endpoints
 ///
+// periphery:ignore
```

Comment:
Temporary periphery ignores so I don't have to update `PaymentRemote`, is possible that we can remove these as well.